### PR TITLE
Make the STRIMZI_NAMESPACE change in the Quicksrt guide a bit more clear

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Documentation

### Description

This PR tries to make the Quickstrat bit more clear based on the feedback we got in #7381 -> it lists from what value should the `STRIMZI_NAMESPACE` be changed to what value.

### Checklist

- [x] Update documentation

### CODE REVIEW

**Review by ppatierno — APPROVED** [review]
LGTM but tbh it was clear to me even before. You are actually already providing the section ...
```yaml
env:
- name: STRIMZI_NAMESPACE
  value: my-kafka-project
# ...
```
which shows how you should set the env var. Don't understand why we should show from where we are moving ...

**Comment by scholzj** [comment]
> LGTM but tbh it was clear to me even before. You are actually already providing the section ...

Well, it was clear to me as well. But we are not the target group, so if this makes it more clear to new users, it might be worth it..

File: documentation/modules/quickstart/proc-installing-product.adoc [hunk 1]
```diff
@@ -52,6 +52,20 @@ kubectl create ns my-kafka-project
 ----
 
 . Edit the `install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml` file and set the `STRIMZI_NAMESPACE` environment variable to the namespace `my-kafka-project`.
+You can do that by changing the environment variable configuration from:
```
Comment (PaulRMellor) [inline documentation/modules/quickstart/proc-installing-product.adoc:55]:
```suggestion
Change the environment variable configuration from this:
```

File: documentation/modules/quickstart/proc-installing-product.adoc [hunk 2]
```diff
@@ -52,6 +52,20 @@ kubectl create ns my-kafka-project
...
+# ...
+env:
+- name: STRIMZI_NAMESPACE
+  valueFrom:
+    fieldRef:
+      fieldPath: metadata.namespace
+# ...
+----
++
+To:
```
Comment (PaulRMellor) [inline documentation/modules/quickstart/proc-installing-product.adoc:68]:
```suggestion
To this:
```

**Review by PaulRMellor — APPROVED** [review]

**Pull request merged**